diff --git a/Makefile b/Makefile
index 62ee55b..5036b28 100644
--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,7 @@
-
-	# check if user is root
-	user = $(shell whoami)
-	ifeq ($(user),root)
-	INSTALL_DIR = /usr/lib/lv2
-	else 
-	INSTALL_DIR = ~/.lv2
-	endif
+	INSTALL_DIR = /zynthian/zynthian-plugins/lv2/
 
 	ifneq (cat /proc/cpuinfo | grep sse2 >/dev/null,)
-	SSE_CFLAGS = -msse2 -mfpmath=sse
+	SSE_CFLAGS =
 	else ifneq (cat /proc/cpuinfo | grep sse >/dev/null,)
 	SSE_CFLAGS = -msse -mfpmath=sse
 	else
