diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6114080..af58f88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,8 +25,8 @@ link_directories   ( ${NTK_LIBRARY_DIRS}  )
 SET(CMAKE_SHARED_LINKER_FLAGS "-fPIC -shared -Wl,-z,nodelete -Wl,--no-undefined")
 
 IF(RELEASE_BUILD)
-  SET(CMAKE_CXX_FLAGS "-g -Wall -Wno-unused-variable -msse2 -mfpmath=sse -ffast-math")
-  SET(CMAKE_C_FLAGS "-g -Wall -W -Wno-unused-variable -msse2 -mfpmath=sse -ffast-math")
+  SET(CMAKE_CXX_FLAGS "-g -Wall -Wno-unused-variable -ffast-math -fpermissive")
+  SET(CMAKE_C_FLAGS "-g -Wall -W -Wno-unused-variable -ffast-math -fpermissive")
 ENDIF()
 
 ## The following commands will re-generate the .cpp files from the FAUST .dsp,
