diff --git a/Makefile b/Makefile
index eb94b0e..5c6fae8 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ LIBRARY = libsosynth.so
 TTLS = so-666.ttl so-kl5.ttl so-404.ttl manifest.ttl
 CC = gcc
 CFLAGS += -Wall -O3 -ffast-math -lm `pkg-config --cflags --libs lv2` -fPIC
-INSTALLDIR = $(DESTDIR)/usr/lib/lv2/
+INSTALLDIR = $(DESTDIR)/zynthian/zynthian-plugins/lv2/
 INSTALLNAME = so-synth.lv2/
 $(LIBRARY) : $(OBJECTS)
 	$(CC) $(CFLAGS) $(OBJECTS) -shared -o $@
