diff --git a/src/faust-dsp-plugin.cpp b/src/faust-dsp-plugin.cpp
index 17a7dfb..882294f 100644
--- a/src/faust-dsp-plugin.cpp
+++ b/src/faust-dsp-plugin.cpp
@@ -49,7 +49,7 @@ class mydsp_ex : public mydsp
 {
 public:
 	mydsp_ex()
-		{ data.osc = NULL; }
+		{ data.osc = 0; }
 
 	mydsp_user_data data;
 };
diff --git a/src/faust-dsp-standalone.cpp b/src/faust-dsp-standalone.cpp
index d38c33c..57f10d8 100644
--- a/src/faust-dsp-standalone.cpp
+++ b/src/faust-dsp-standalone.cpp
@@ -49,7 +49,7 @@ class mydsp_ex : public mydsp
 {
 public:
 	mydsp_ex()
-		{ data.osc = NULL; }
+		{ data.osc = 0; }
 
 	mydsp_user_data data;
 };
