diff --git a/include/foo-yc20.h b/include/foo-yc20.h
index ce278d3..61dbff0 100644
--- a/include/foo-yc20.h
+++ b/include/foo-yc20.h
@@ -33,7 +33,7 @@ ADVISEDOF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include <map>
 #include <string>
-#include <faust/audio/dsp.h>
+#include <faust/dsp/dsp.h>
 #include <faust/gui/UI.h>
 
 #include <faust-dsp.h>
diff --git a/src/faust-dsp-plugin.cpp b/src/faust-dsp-plugin.cpp
index 17a7dfb..b26ec0e 100644
--- a/src/faust-dsp-plugin.cpp
+++ b/src/faust-dsp-plugin.cpp
@@ -41,7 +41,7 @@ inline float min(float x, float y)
 }
 
 #include "faust/gui/UI.h"
-#include "faust/audio/dsp.h"
+#include "faust/dsp/dsp.h"
 #include "faust/gui/meta.h"
 #include "../gen/yc20-dsp-plugin.cpp"
 
diff --git a/src/faust-dsp-standalone.cpp b/src/faust-dsp-standalone.cpp
index d38c33c..add81ab 100644
--- a/src/faust-dsp-standalone.cpp
+++ b/src/faust-dsp-standalone.cpp
@@ -41,7 +41,7 @@ inline float min(float x, float y)
 }
 
 #include "faust/gui/UI.h"
-#include "faust/audio/dsp.h"
+#include "faust/dsp/dsp.h"
 #include "faust/gui/meta.h"
 #include "../gen/yc20-dsp-standalone.cpp"
 
