diff --git a/Makefile.mk b/Makefile.mk
index 214c8be..6d60b76 100644
--- a/Makefile.mk
+++ b/Makefile.mk
@@ -23,12 +23,7 @@ endif
 # Set build and link flags
 
 BASE_FLAGS = -Wall -Wextra -pipe
-BASE_OPTS  = -O2 -ffast-math -mtune=generic -msse -msse2 -fdata-sections -ffunction-sections
-
-ifneq ($(MACOS),true)
-# MacOS doesn't support this
-BASE_OPTS += -mfpmath=sse
-endif
+BASE_OPTS  = -O2 -ffast-math -mtune=generic -fdata-sections -ffunction-sections
 
 ifeq ($(MACOS),true)
 # MacOS linker flags
